tests/php/Model/SiteTreeTest.php

```diff
@@ -1223,6 +1230,11 @@ public function testPageTypeClasses()
             $newClasses,
             'Setting hide_ancestor to a boolean (incorrect) value caused a page class to be hidden'
         );
+
+        // Testing what happens if a valid config value is set
+        Config::modify()->set(SiteTreeTest_ClassA::class, 'hide_ancestor', 'Page');
+        $classes = SiteTree::page_type_classes();
+        $this->assertNotContains('Page', $classes);
```

Comment:
Without this addition, we don't actually have any test coverage validating that the old config even works 😅